src/stories/TextInputWithTokens.stories.tsx

```diff
@@ -30,7 +32,53 @@ export default {
         </ThemeProvider>
       )
     }
-  ]
+  ],
+  argTypes: {
+    maxHeight: {
+      defaultValue: undefined,
+      control: {
+        type: 'text'
+      }
+    },
+    preventTokenWrapping: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    size: {
+      name: 'size (token size)',
+      defaultValue: 'extralarge',
+      options: ['small', 'medium', 'large', 'extralarge'],
+      control: {
+        type: 'radio'
+      }
+    },
+    inputSize: {
+      defaultValue: 'small',
+      options: ['small', 'large'],
+      control: {
+        type: 'radio'
+      }
+    },
+    hideTokenRemoveButtons: {
+      defaultValue: false,
+      control: {
+        type: 'boolean'
+      }
+    },
+    visibleTokenCount: {
+      defaultValue: undefined,
+      control: {
+        type: 'number'
+      }
+    },
+    validationStatus: {
+      options: ['warning', 'error', undefined],
```

Comment:
missing 'success' here?

Reply:
We don't use 'success' for `TextInput`, so I didn't add it for `TextInputWithTokens`.

Reply:
Does it use `TextInputWrapper` under the hood? If so, it does support `success` now in the base component: https://github.com/primer/react/blob/ce7bef34b9a4024ee63ee79da880f9b9aac7f74e/src/_TextInputWrapper.tsx#L46

Reply:
Ok, then let's accept "success" in both `TextInput` and `TextInputWithTokens`.

We're going to be discussing the "warning" and "success" statuses in next week's Primer patterns working session, so we can update if we need to.